.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+## Description
+
+A short summary of the idea.
+
+## How will this impact [SecureDrop users](https://github.com/freedomofpress/securedrop-ux/wiki/Users)?
+<!-- How do you feel this change might impact SecureDrop's usability, accessibility, or usefulness—and specifically, for which users? Has anecdotal feedback from users influenced this change? Does evidence exist from user research to support this idea? Could design or user research efforts be helpful to support a change? -->
+
+## How would this affect SecureDrop's [threat model](https://docs.securedrop.org/en/stable/threat_model/threat_model.html)?
```

Comment:
We could also link to https://github.com/freedomofpress/securedrop-workstation/#threat-model here.

Reply:
Yes, let's link to the securedrop workstation threat model

Reply:
Yes, let's link to the securedrop workstation threat model

Reply:
Adding multiple links in these templates can get a bit unwieldy, so instead I've opened a separate docs PR to make the SDW threat model a bit more discoverable:
https://github.com/freedomofpress/securedrop-docs/pull/302

Reply:
#302 has been merged. I still think it makes more sense for a developer to be directed directly to the SDW threat model, because if you follow the https://docs.securedrop.org/en/stable/threat_model/threat_model.html link, you'll have to read about the old SVS and other pieces that are different in the new architecture.

Reply:
OK, switched the links - that one also links back to the server threat model, in any event.